windows/Dockerfile

```diff
@@ -23,6 +23,8 @@ ENV WINDOWS_VERSION=${WINDOWS_VERSION:-1809}
 ARG DD_TARGET_ARCH
 ENV TARGET_ARCH=${DD_TARGET_ARCH:-x64}
 
+ARG GOROOT
+
```

Comment:
Is this a WIP leftover, or is this still needed for Windows?

Reply:
Still needed for now, value is populated from `build-container.ps1`. I'll modify that to use the same runtime logic as the linux images but for now I want to unblock the datadog-agent PR